src/Core/ControllerFactory.php

```
@@ -154,20 +154,28 @@ public function make(string $controllerClass): object
         if ($controllerClass === UserStoryController::class) {
             $storyAiServices = new StoryAiServiceFactory($runtimeConfig);
             $storyQualityScorer = $storyAiServices->createQualityScorer();
+            $contextPromptPacketBuilder = new ContextAwarePromptPacketBuilder(
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $contextPromptPacketBuilder. Keep variable name length under 20. (undefined)

(LongVariable)